offix-web/gulpfile.js

```diff
@@ -0,0 +1,78 @@
+// PACKAGES //
```

Comment:
can we get rid of the gulpfile? Most of the stuff in here is unnecessary.

The only thing that looks necessary is the scss compilation. Instead of ahead-of-time compilation, can we use this: https://www.npmjs.com/package/node-sass-middleware ?